### PR TITLE
test utils

### PR DESCRIPTION
**Pull request by JackCrumpLeys**
unify test data for better deterministic benchmarks and testing.

### CODE REVIEW

File: crates/index/benches/ray.rs [hunk 1]
```diff
@@ -87,8 +65,8 @@ fn setup_world(world: &mut World, num_entities: u32, max_distance: f32) {
     }
 
     let mut rays = Rays::new();
-    let ray_origins = load_points(1000);
-    let ray_dirs = load_points(1000);
+    let ray_origins = load_points(&1000.try_into().unwrap(), MAP_SIZE);
```
Comment (JackCrumpLeys) [inline crates/index/benches/ray.rs:68]: should we just use the enum variant here?

Comment (Indy2222) [reply crates/index/benches/ray.rs:68]: I would.

File: crates/test_utils/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            100_000 => Ok(Self::OneHundredThousand),
+            _ => Err("Invalid number of points"),
+        }
+    }
+}
+
+impl From<NumPoints> for usize {
+    fn from(value: NumPoints) -> Self {
+        match value {
+            NumPoints::OneHundred => 100,
```
Comment (Indy2222) [inline crates/test_utils/src/lib.rs:33]: I would replace this with `Self::from(&value)` so that it stays DRY.

**Review by Indy2222 — CHANGES_REQUESTED** [review]
Two minor notes, otherwise LGTM.

**Review by Indy2222 — APPROVED** [review]

**Pull request merged**